### ISSUE TITLE
Nightguard no longer notifies when the sensor needs calibration or is updating/starting up

### ISSUE THREAD

**Issue by JorgeLozanoConesa** [nightscout/nightguard#179]
Hello,

First of all thank you for the great work you are doing with Nightguard.

I am contacting you to inform you of an issue that I am suffering since the last update of the app, a couple of months ago and that does not allow me to trust the alerts for missing data as before.

**Currently, Nightguard no longer notifies me when the sensor needs calibration or is updating/starting up. I used to set the "dummy" reading of 5 or 2 and the alert would sound. Now it displays the last value read before calibration and no longer alerts or alarms.  
I attach screenshots from this morning, after spending all night without signal.
(since I can't help you programming, I'll at least help you testing😉)**

Thanks in advance and congratulations for the work you do.
Jorge

Pump/Sensor -> Medtronic 640G/Smartguard
Nightscout in Heroku

![nightguard1](https://user-images.githubusercontent.com/43475073/151316682-4aec8ac2-d7e0-491c-a21b-c4097cea3703.png)
![NIghtguard2](https://user-images.githubusercontent.com/43475073/151316687-8bf9153c-5a83-4975-88b0-3e729f95e0c8.png)
![NIghtguard3](https://user-images.githubusercontent.com/43475073/151316695-7b7d1e37-5a0b-4451-b9c0-b62a4ecbe4e7.png)

**Comment by dhermanns** [comment]
Thanks for your Feedback. Looks like the same Problem as #175 to me. Did you test the latest Testflight?
I already changed something there...

**Comment by JorgeLozanoConesa** [comment]
Hi,
Thank you for your quick response.
I read 175 issue a few days ago, but not the most recent comments.
I just installed TestFlight and the latest beta version and I'll check it out tonight (or with next calibration).
BR

**Comment by dhermanns** [comment]
And I will double check the modifications. Looks like there are still issues that could arise.

**Comment by JorgeLozanoConesa** [comment]
Hi,
With the Beta version (Testflight), the app detect "missed readings" (and sensor calibration and update status) again !!!. 
It used to work differently than previous versions. Before the app would identify a glucose value of 2 (updating) or 5(calibrating) and the "low bg alert" switch on and now, the "missed readings alert" switch on in all three cases, but it works perfectly from my point of view.
Thank you

**Comment by dhermanns** [comment]
Great to hear that. I switched to a newer nightscout endpoint. And the handling of certain values is there a bit different. That caused the different behavior in nightguard ;-)

**Comment by tast3rmonial** [comment]
Dear ,

We are excited to give you an update on Tastermonial.

Today we have initiated a way to evaluate food claims and for customers to get a prescription for continuous glucose monitor (CGM) sensors and experiment with different foods using our app. 

We are actively working to improve your user experiences in:

👉 Ordering a CGM

We realized the uncertainty of CGM prices in major pharmacies around the US. Therefore, our team is now working on a solution to order CGM online and ship directly to you at a competitive price.

👉 Learning from your CGM food experiment data

[See video of current app's abiity](https://youtu.be/xtaoSxdqq6U)

Here's a peek into upcoming Tastermonial app's upgrade in the next few weeks:

- the ability to run food pairing experiments wtih CGM
- automatic syncing of CGM data to view and share the blood sugar experiments
- You'll be able to use our app to scan and evaluate more foods in your pantry (we're expanding our food database)
 We'd be happy to gather your feedback that could help us in prioritizing the next launches. Click the button below to let us know your needs.

[Customer ***@***.***&firstname=&lastname=)

 

Thanks for reading!

Bude & Tastermonial Team

[Facebook](https://www.facebook.com/tastermonial)
[Instagram](https://www.instagram.com/tastermonial)

No longer want to receive these emails? [Unsubscribe](https://manage.kmail-lists.com/subscriptions/unsubscribe?a=TvyBXH&c=01FTD937BTA585DASNWBSJSA9X&k=db51a5ebf667c446259346867b0d1a46&m=WUxwcT&r=RaxVK3m).
TASTERMONIAL
